nextflow.config

```diff
@@ -297,56 +305,85 @@ env {
 }
 
 // Set bash options
-process.shell = """\
-bash
-
-set -e # Exit if a tool returns a non-zero status/exit code
-set -u # Treat unset variables and parameters as an error
-set -o pipefail # Returns the status of the last command to exit with a non-zero status or zero if all successfully execute
-"""
+process.shell = [
+    "bash",
+    "-C",         // No clobber - prevent output redirection from overwriting files.
```

Comment:
We didn't have the -C before, could this be causing the current CI issues?

Reply:
Thanks @maxulysse ! That was it. We had a local module re-using a temp file name and causing the clobber incompatibility, so I've taken that out. Fingers crossed.